packages/grid/x-data-grid/src/hooks/features/clipboard/useGridClipboard.ts

```diff
@@ -25,6 +25,22 @@ function writeToClipboardPolyfill(data: string) {
   }
 }
 
+function hasNativeSelection() {
+  if (window.getSelection()?.toString() !== '') {
+    return true;
+  }
+
+  const activeElement = document.activeElement as HTMLInputElement;
```

Comment:
Thanks, @m4theushw!

I believe we should also use `getRootNode` instead of `ownerDocument` here:
https://github.com/mui/mui-x/blob/2c282eaaa7659fa7cef66bb17e3f5233bf360355/packages/grid/x-data-grid/src/components/cell/GridCell.tsx#L189
Correct?

Reply:
As for `document.activeElement`, it looks like I can just use `event.target` instead to simplify things.